ui/perfherder/perf-helpers/helpers.js

```diff
@@ -800,9 +800,9 @@ export const createGraphData = (
         alertSummary: alertSummaries.find(
           (item) => item.push_id === dataPoint.push_id,
         ),
-        commonAlert: commonAlerts.some((item) => {
-          return item.find((alert) => alert.push_id === dataPoint.push_id);
-        }),
+        commonAlertSummary: commonAlerts[0].find(
```

Comment:
why do we search here only in first element?

Reply:
because `commonAlertSummary` is an array with elements only on position 0